backend/src/main/java/ch/nova_omnia/lernello/api/CreateLearningKitController.java

```diff
@@ -0,0 +1,33 @@
+package ch.nova_omnia.lernello.api;
+
+import ch.nova_omnia.lernello.dto.request.CreateLerningKitDTO;
+import ch.nova_omnia.lernello.dto.response.LearningKitResDTO;
+import ch.nova_omnia.lernello.mapper.LearningKitMapper;
+import ch.nova_omnia.lernello.repository.LearningKitRepository;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/learning-kits") //TODO: what does this do or where do i have to implement this??
```

Comment:
this acts like a namesspace for all mappings sinide this controller. so basically any endpoint inside will need to be prefixed with this